README.md

```diff
@@ -109,7 +109,6 @@ Kubernetes questions & development:
 
 Check out the [list of companies](./ADOPTERS.md) already using Podman Desktop.
 
-
```

Comment:
I think this line can stay as it was before (as there is nothing related to this PR)

Reply:
it's an artefact after running `yarn format:fix`, will revert it manually